updater/osv/osv.go

```diff
@@ -538,14 +539,29 @@ func (e *ecs) Insert(ctx context.Context, name string, a *advisory) (err error)
 							Msg("unsure how to interpret event")
 					}
 				case `ECOSYSTEM`:
-					switch {
-					case ev.Introduced == "0": // -Inf
-					case ev.Introduced != "":
-					case ev.Fixed != "":
-						v.FixedInVersion = ev.Fixed
-					case ev.LastAffected != "":
-					case ev.Limit == "*": // +Inf
-					case ev.Limit != "":
+					switch af.Package.Ecosystem {
+					case ecosystemGo:
+						panic("ecosystem: go")
+					case ecosystemMaven:
+						switch {
+						case ev.Introduced == "0":
+						case ev.Introduced != "":
+							v.FixedInVersion = ev.Introduced + "+"
+						case ev.Fixed != "":
+							v.FixedInVersion += ev.Fixed
+						case ev.LastAffected != "":
+							v.FixedInVersion += "LastAffected:" + ev.LastAffected
+						}
```

Comment:
I'm unsure what I think about this version packing scheme.

Reply:
> I'm unsure what I think about this version packing scheme.

I couldn't think of a better way to store version range, maybe we can have one more field in `Vulnerability struct` ? I guess that will require db change.

Reply:
I think this should use url.Values instead of a custom little string format, here.